therapy/main.py

```diff
@@ -98,14 +104,19 @@ def search(q: str = Query(..., description=q_descr),
          response_description=merged_response_descr,
          response_model=NormalizationService,
          description=normalize_description)
-def normalize(q: str = Query(..., description=merged_q_descr)) -> Dict:
-    """Normalize query or concept ID provided by user. Return strongest-match merged
-    concept.
+def normalize(q: str = Query(..., description=merged_q_descr),
+              infer_namespace: Optional[bool] = Query(
```

Comment:
essentially the same thing
```suggestion
              infer_namespace: bool = Query(
```

Reply:
Oh this is very true